go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/DataDog/go-libddwaf/v2
 
-go 1.18
+go 1.19
```

Comment:
Has nothing to do in this PR, same for the filename change but we can let it slide I guess